backend.py

```diff
@@ -426,6 +426,13 @@ def add_repos(main_repositories, update_repositories, repos):
     if not main_repositories or main_repositories[0].identifier() != MAIN_REPOSITORY_NAME:
         raise RuntimeError("No main repository found")
 
+    if 'gpgcheck' in answers and not answers['gpgcheck']:
```

Comment:
Why not directly `not answers.get('gpgcheck')`?

Reply:
Just hesitation on my side - you're right it's just better

Reply:
In fact there was a reason: `get()` returns `None` when the parameter was never specified, which would be taken as `False`, whereas the default we want is `True`.  Will add a comment to make this more explicit.

Reply:
I suppose we could use `get(..., True)`.

Reply:
right, that's better